ndcube/ndcube.py

```diff
@@ -554,14 +564,14 @@ def axis_world_coords_values(self, *axes, pixel_corners=False, wcs=None):
         CoordValues = namedtuple("CoordValues", identifiers)
         return CoordValues(*axes_coords[::-1])
 
-    def crop(self, *points, wcs=None):
+    def crop(self, *points, wcs=None, keepdims=False):
```

Comment:
Is there utility in making either this or everything to be enforced as keywords arguments?

Reply:
I don't see the need to be that restrictive.  Unless there's a good argument for it?

Reply:
Well if you ever see the future of where you keep adding keywords, the ordering matters and people will assume that order for you. E.g plot in sunpy.map first arg not being ax for example.

I would personally go on the side of restrictive but its your library.

Reply:
This is already kwarg only @nabobalis because of the `*points`.